diesel_derives/src/lib.rs

```diff
@@ -22,8 +22,10 @@ extern crate proc_macro2;
 extern crate quote;
 extern crate syn;
 
-use proc_macro::TokenStream;
-use syn::parse_macro_input;
+use {
+    proc_macro::TokenStream,
+    syn::{parse_macro_input, parse_quote},
+};
```

Comment:
I would prefer to stay with non-nested imports here, just to be consistent with the rest of the code base.

Reply:
I have some more of that in the dsl_auto_type crate:
https://github.com/Ten0/diesel/blob/71a5a4236e4dd860b1e0dd4ccfe42481b4b11743/dsl_auto_type/src/auto_type/mod.rs#L6-L13

Is that ok there or do you have a strong preference for changing it there as well?

Reply:
As long as it is consistent in a single crate I do not have strong preferences here.